python/lib/bidsreader.py

```diff
@@ -48,8 +49,11 @@ def __init__(self, bids_dir):
 
         # load dataset name and BIDS version
         dataset_json = bids_dir + "/dataset_description.json"
-        self.dataset_name = self.bids_layout.get_metadata(dataset_json)['Name']
-        self.bids_version = self.bids_layout.get_metadata(dataset_json)['BIDSVersion']
+        dataset_description = {}
+        with open(dataset_json) as json_file:
+            dataset_description = json.load(json_file)
+        self.dataset_name = dataset_description['Name']
+        self.bids_version = dataset_description['BIDSVersion']
```

Comment:
with the new version of `pybids`, there is no way to extract data from the `dataset_description.json` file so using the `json` python library to parse the file.